src/components/delegates/delegates.js

```diff
@@ -54,6 +56,39 @@ app.component('delegates', {
       this.$scope.$on('peerUpdate', () => {
         this.updateAll();
       });
+
+      const $ctrl = this;
+
+      this.infiniteDelegates = {
+        getItemAtIndex(index) {
+          if (index > $ctrl.delegates.length) {
+            $ctrl.loadMore();
+            return null;
+          }
+          return $ctrl.delegates[index];
+        },
+        getLength() {
+          return $ctrl.delegatesTotalCount;
+        },
+      };
+    }
+
+    /**
+     * Sets height of the virtual repeat container based on window height and
+     * virtual repeat sizer height (depends on this.delegates.length)
+     *
+     * @method setVirtualRepeatContainerHeight
+     */
+    setVirtualRepeatContainerHeight() {
+      const setHeight = () => {
+        const element = angular.element('delegates md-virtual-repeat-container');
+        const repeatSizerHeight = angular.element('.md-virtual-repeat-sizer').height();
+        const newHeight = Math.round(Math.min(
+          repeatSizerHeight + 30,
+          this.$window.innerHeight * 0.7));
+        angular.element(element).css('height', `${newHeight}px`);
```

Comment:
The default unit in `css` method is `px`, so you don't need this string template.